configuration/concepts/1300_services.csv

```diff
@@ -0,0 +1,25 @@
+Uuid,Description:en,Answers,Fully specified name:fr,Fully specified name:en,Short name:fr,Short name:en,Data class,Data type,_order:1300
```

Comment:
@lauravignoli shall we use the following translations for `Short name:fr` french? (got the translations from the deleted keys in the config repository) 
```
"ANC" => "CPN"
"ART" => "UPEC"
"CARDIOLOGY" => "Cardiologie"
"DELIVERY" => "D'accouchement"
"DENTISTRY" => "Dentisterie"
"EMERGENCY" => "Urgence"
"GYNAECOLOGY" => "Gynécologie"
"LABORATORY" => "Laboratoire"
"MATERNITY" => "Maternite"
"MINOR_SURGERY" => "Petite Churigie"
"OPD" => "Consultations Externes"
"OPHTHALMOLOGY" => "Ophtalmologie"
"PEDIATRIC" => "Pediatrie"
"RADIOLOGY" => "Radiologie"
"KINESIOLOGY" => "Kinésiologie"
"MAJOR_SURGERY" => "Grande Chirurgie"
"NEONATOLOGY" => "Ajouter neonatologie"
"POST_NATAL" => "Post Natal"
"PSYCHOLOGY" => "Physiothérapie"
"RHEUMATOLOGY" => "Rhumatologie"
"FAMILYPLANNING" => "Planning familial"
"PHYSIOTHERAPY" => "Physiothérapie"
"ART_DISPENSATION" => "UPEC Dispensation"
```

Reply:
Well spotted, I forgot to add them.